packages/medusa/src/services/totals.ts

```diff
@@ -703,7 +703,7 @@ class TotalsService extends TransactionBaseService {
     cartOrOrder: Cart | Order,
     discount: Discount
   ): LineDiscountAmount[] {
-    let merged: LineItem[] = [...cartOrOrder.items]
+    let merged: LineItem[] = cartOrOrder.items ? [...cartOrOrder.items] : []
```

Comment:
Adding resilience to method when cart.items is undefined.

Reply:
[…(cartOrOrde.items ?? [])] but i think you should not bother as it has changed in my pr

Reply:
I don't see the added value. Why spread an empty array. For info, here's the transpiled for both:
```javascript
// let merged: LineItem[] = cartOrOrder.items ? [...cartOrOrder.items] : []
var merged = cartOrOrder.items ? __spreadArray([], __read(cartOrOrder.items), false) : [];

// let merged: LineItem[] = [...(cartOrOrder.items ?? [])]
var merged = __spreadArray([], __read(((_c = cartOrOrder.items) !== null && _c !== void 0 ? _c : [])), false);
```

Reply:
there is not it is only a suggestion that you can take or leave. Only for consistency, and I know the transpiled code as I am working in it 😂

Reply:
For consistency, then I can get onboard. Somethings weird though, your suggestion disappeared. Was that intentional or is there a bug in GitHub?
<img width="851" alt="Screen Shot 2022-11-14 at 9 56 14 AM" src="https://user-images.githubusercontent.com/116003638/201692614-190745ad-4a22-41e4-ba47-27264b2d5cee.png">

Reply:
it was intended to not loose time on that as it is really nit non blocking 😂

Reply:
Now it looks like I'm talking to myself.
<img width="765" alt="Screen Shot 2022-11-14 at 10 03 37 AM" src="https://user-images.githubusercontent.com/116003638/201693797-8a488743-c173-4da7-9c35-0c20a7f86ff0.png">
^ this guy is nuts

Reply:
🤣 it was not displayed like this for me but as a standalone message oups, no worries it is resolved